### DISCUSSION TITLE
Webpack errors when i build project after updating NodeJS to 18.13.0 and Docusaurus 2.3.0

### DISCUSSION THREAD

**Discussion by MiamiVince33** [facebook/docusaurus#8626] [Q&A]
Hello,

I have a Docusaurus project for my Team at work and i updated Node JS to 18.3.0 and Docusaurus to 2.3.0. I update the versions regularly so I didn't have a big gap with the previous one but when I build the project, i have errors that seem to be related to the webpack present in nodes_modules. This is the first time I had problems to build the project since I created my Docusaurus 1 year and half ago. 

Do you have any idea how to solve the problem ?

Thank you in advance :) 

![Erreur docusaurus](https://user-images.githubusercontent.com/124350641/216568787-776d3023-3f44-4695-a90e-97d792c3db74.PNG)

**Comment by slorber** [comment]
It's very difficult to help without a repro. 

Please init a new docusaurus site see if a fresh site works under 2.3 + Node 18.3, it should. Then try adding your site specific things until you can reproduce the problem. You can also try to remove your site features one by one until it starts working again.

If you are in a monorepo, double check your dependencies to see how they changed. 

Maybe upgrade to 2.2.x first and see where it stopped working exactly because if you upgraded multiple version at once we can't know when it stopped working.

It looks related to the npm `chalk` package and module resolution. Double-check that you didn't update it by mistake to v5 (ES modules only) because we only support v4 (CJS).
